tests/drivers/counter/counter_basic_api/src/test_counter.c

```diff
@@ -254,23 +254,33 @@ void test_set_top_value_without_alarm(void)
 static void alarm_handler(struct device *dev, u8_t chan_id, u32_t counter,
 			  void *user_data)
 {
+	/* Arbitrary limit for alarm processing - time between hw expiration
+	 * and read-out from counter in the handler.
+	 */
+	static const u64_t processing_limit_us = 1000;
```

Comment:
Why static? Just curious.

Reply:
puts variable in flash and not on stack.

Reply:
Are we really so close to blowing the stack on this simple test case? I'm not objecting to it, it just looked weird.